SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialNetworkingMap.cpp

```diff
@@ -58,6 +58,7 @@ void USpatialNetworkingMap::CreateCustomContentForMap()
 	AddActorToLevel<ADynamicActorAwakeAfterDormantChangePropertyTest>(CurrentLevel, FTransform::Identity);
 
 	AddActorToLevel<AUnresolvedReferenceTest>(CurrentLevel, FTransform::Identity);
```

Comment:
very minor: churn

Reply:
churn?

Reply:
code churn AKA (white space) changes that don't change anything but do show up in git polluting the history.

Reply:
oh sure, can remove

Reply:
is gone now